container/image.bzl

```diff
@@ -221,15 +206,14 @@ def _impl(ctx, files=None, file_map=None, empty_files=None, directory=None,
   layer_diff_ids = [layer.diff_id for layer in layers]
   diff_ids = parent_parts.get("diff_id", []) + layer_diff_ids
 
-  # Get and merge environment variables
-  env = {}
-  [_evalute_merge_env(env, layer.env) for layer in layers]
-  # Generate the new config using the attributes specified and the diff_id
-  config_file, config_digest = _image_config(
-      ctx, layer_diff_ids,
-      entrypoint=entrypoint or ctx.attr.entrypoint,
-      cmd=cmd or ctx.attr.cmd,
-      env=env)
+  # Get the config for the base layer
+  config_file = _get_base_config(ctx)
+  # Generate the new config layer by layer, using the attributes specified and the diff_id
+  for i in range(len(layers)):
```

Comment:
for i, layer in enumerate(layers):

Reply:
Done. 

Many thanks for the review and approval!!